core/modules/widgets/link.js

```diff
@@ -39,7 +39,7 @@ LinkWidget.prototype.render = function(parent,nextSibling) {
 		useWikiLinks = wikiLinksMacro ? (wikiLinksMacro.trim() !== "no") : true,
 		missingLinksEnabled = !(this.hideMissingLinks && this.isMissing && !this.isShadow);
 	// Render the link if required
-	if(useWikiLinks && missingLinksEnabled) {
+	if(useWikiLinks && missingLinksEnabled || this.showLinks) {
```

Comment:
I'm not so happy with the name. ... For me it is more like (lacking a better name) "forceLinks" ...

Reply:
or may be "enforceLinks"

Reply:
no problem, I was just lacking a better one

enforceLinks sounds good

Reply:
or retainLinks or keepLinks
someone other than me should decide that